apt-hook/Makefile

```diff
@@ -1,3 +1,21 @@
+# the go hook steps will be skipped if this is a nonempty string
+SKIP_GO_HOOK = ""
+
+GO_BIN = $(shell which go)
+# Try go-1.14 and go-1.10 if not in path
+ifeq ($(GO_BIN),)
+	ifneq ($(wildcard /usr/lib/go-1.14/bin/go),)
+		GO_BIN = /usr/lib/go-1.14/bin/go
+	else ifneq ($(wildcard /usr/lib/go-1.10/bin/go),)
```

Comment:
Why are we looking at go-1.10 here instead of go-1.13 per what we are requiring in d/control?

Reply:
Shoot, I forgot to update this after the discussion with Paride resulted in a change to the dependency line. I will fix this